addon/components/tf-toggle-button.js

```diff
@@ -0,0 +1,69 @@
+/**
+  @module ember-ticketfly-toggle-button
+ */
+import Component from 'ember-component';
+import layout from '../templates/components/tf-toggle-button';
+import get from 'ember-metal/get';
+import computed from 'ember-computed';
+
+/**
+  colorGroup options: 'neutral'
+  sizeGroup options: 'primary', 'secondary'
+
+  @public
+  @class TfToggleButton
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+
+  tagName: 'button',
+  type: 'button',
+  attributeBindings: ['disabled', 'aria-label', 'aria-pressed', 'type'],
+  classNameBindings: ['buttonCategory', 'disabled:c-tf-toggle-button--disabled', 'pressed:c-tf-toggle-button--pressed'],
+  classNames: ['c-tf-toggle-button', 'u-pv-0', 'u-ph-xs'],
+
+  hook: 'tf-toggle-button',
+
+  onClick() { return this; }, //noop
+
+  click() {
+    this.toggleProperty('pressed');
+    const pressed = get(this, 'pressed');
+    get(this, 'onClick')(pressed);
```

Comment:
Also — is `onClick` the right action name for this? That implies that the argument that gets passed will be the click event...perhaps we should just support `action`?